src/tests/multihost/admultidomain/test_multiforest.py

```diff
@@ -11,61 +11,58 @@
 class TestADMultiForest(object):
 
     @staticmethod
-    def test_0001_multiforest(multihost, newhostname, adjoin):
+    def test_0001_multiforest(self, multihost, newhostname, adjoin):
         """
-        :title: IDM-SSSD-TC: ad_provider: admultiforest
-        :id:
+        :title: IDM-SSSD-TC: ad_provider: admultiforest : Authentication against two forests
+        :id: 900f2467-1aca-430c-bbaa-b22d30a829ad
         :setup:
           1. Configure two domain controllers in different forests
           2. Join client to the first domain
           3. Update sssd.conf for second domain
           4. Update krb5.conf for second domain
-          5. Create krb principal and update sssd.conf
+          5. Create krb principal and keytab for second domain in the second forest and update sssd.conf
         :steps:
-          1. Lookup user in the first domain
-          2. Lookup user in the second domain
+          1. Lookup user in the root domain of the first forest
+          2. Lookup user in the  root domain fo the second forest
         :expectedresults:
-          1. User is found in the first domain
-          2. User is found in the second domain
+          1. User is found in the root domain in the first forest
+          2. User is found in the root domain in the second forest
         :customerscenario: True
         """
         adjoin(membersw='adcli')
         ad_domain = multihost.ad[0].domainname
         ad_server = multihost.ad[0].hostname
-        # This must be the last AD server in the metadata file
-        ad1_domain = multihost.ad[len(multihost.ad) - 1].domainname
+        # The second forest domain must be the last entry in the metadata file
+        ad1_ip = multihost.ad[len(multihost.ad)-1].ip
+        ad1_domain = multihost.ad[len(multihost.ad)-1].domainname
         ad1_domain_upper = str.upper(ad1_domain)
-        ad1_server = multihost.ad[len(multihost.ad) - 1].hostname
-        ad1_password = multihost.ad[len(multihost.ad) - 1].ssh_password
+        ad1_server = multihost.ad[len(multihost.ad)-1].hostname
+        ad1_password = multihost.ad[len(multihost.ad)-1].ssh_password
 
-        get_keytab = f'adcli join --host-keytab=/etc/krb5.keytab-domain1 ' \
-                     f'{ad1_domain}'
+        multihost.client[0].run_command('cp -rf /etc/hosts /etc/hosts.bak')
+        multihost.client[0].run_command(f'echo "{ad1_ip}    {ad1_server}" >>  /etc/hosts')
```

Comment:
Extending the previous comment, you can use the content variable from before to load new data before writing back to /etc/hosts:

```
new_content = content + f'{ad1_ip}    {ad1_server}'
multihost.client[0].put_file_contents('/etc/hosts', new_content)

```